CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
 - Add warning when a developer is using yarn@2 PnP (#2198).
 - Fixes incorrect URLs reported inside emulated HTTPS functions (#1862).
 - Updates underlying timeout handler when proxying through the Hosting emulator.
+- Add support for deploying [experimental extensions](https://github.com/FirebaseExtended/experimental-extensions) (#2398) 
```

Comment:
```suggestion
- Add support for installing [experimental extensions](https://github.com/FirebaseExtended/experimental-extensions) (#2398) 
```

suggesting this to match the language we use in the [docs](https://firebase.google.com/docs/extensions/install-extensions)